### DISCUSSION TITLE
What Does CP Use to Measure Length/Width of Attachments?

### DISCUSSION THREAD

**Discussion by lcc1211** [Courseplay/Courseplay_FS22#2690] [Q&A]
This applies to vanilla and MOD equipment. Giant says the Case IH 3162 TerraFlex Draper header is 13.7 meters, but CP says 13.4. It is actually 13.7 as I've manually adjusted it and not missed any bits. There other MODs but this is an example.

**Comment by Tensuko** [comment]
Depends on tool.
AI Markers, workArea, or directly configured.
You can always put your own width in a user config.
